src/templates/_common/components/forms.jsx

```diff
@@ -44,14 +44,38 @@ export const FormRow = ({
                         const option_id = `${id}_option_${i}`;
                         return (
                             <div key={value}>
-                                <input type='radio' name={id} id={option_id} value={value} defaultChecked={value === default_option} />
-                                <label htmlFor={option_id}><span data-balloon={data_balloon_text} data-balloon-length='medium'>{label_text}</span></label>
+                                <input
+                                    type='radio'
+                                    name={id}
+                                    id={option_id}
+                                    value={value}
+                                    defaultChecked={value === default_option}
+                                />
+                                <label htmlFor={option_id}>
+                                    <span
+                                        data-balloon={data_balloon_text}
+                                        data-balloon-length='medium'
+                                    >{label_text}
```

Comment:
as far as I know we don't have a convention for putting the closing tag and text of the element on the same line.